lib/algos/list/MPU.js

```diff
@@ -34,6 +35,28 @@ class MultipartUploads {
         this.logger = logger;
     }
 
+    genMDParams() {
+        const params = {};
+        if (this.params.keyMarker) {
+            params.gt = `overview${this.params.splitter}` +
+                `${this.params.keyMarker}${this.params.splitter}`;
+            if (this.params.uploadIdMarker) {
+                params.gt += `${this.params.uploadIdMarker}`;
```

Comment:
I think the `+` operator on a string automatically converts its second argument into a string.